apiv2/views.py

```diff
@@ -54,14 +54,14 @@
     from collections import OrderedDict
 except:
     from freesound.utils.ordered_dict import OrderedDict
-from urllib import unquote, quote
+from urllib import quote
```

Comment:
`quote`? `quote` is used. I removed `unquote` because pycharm highlighted it as not used, but in fact these lines are unrelated with the changes in the PR, is one of these code "tidy ups" that I should have probably committed separately but that, hmm, not too bad is it? ;)